app/shared/productTypes.tsx

```diff
@@ -10,28 +10,31 @@ import { membershipCancellationReasons } from "../client/components/cancel/membe
 import { MeValidator } from "../client/components/checkFlowIsValid";
 import { NavItem, navLinks } from "../client/components/nav";
 import { MeResponse } from "./meResponse";
-import { formatDate, Subscription } from "./productResponse";
+import { formatDate, ProductDetail, Subscription } from "./productResponse";
 
 export type ProductFriendlyName =
   | "membership"
   | "recurring contribution" // TODO use payment frequency instead of 'recurring' e.g. monthly annual etc
   | "newspaper subscription"
   | "digital pack"
```

Comment:
This list doesn't feel consistent. If we are using "newspaper subscription", I think it would be more consistent to have "digital pack subscription" and "Guardian Weekly subscription"?

Reply:
done